packages/widgetbook/lib/src/knobs/iterable_segmented_knob.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:meta/meta.dart';
+
+import '../fields/fields.dart';
+import '../fields/iterable_segmented_field.dart';
+import 'knob.dart';
+
+@internal
+class IterableSegmentedKnob<T> extends Knob<Iterable<T>?> {
```

Comment:
We need to add an example of this knob to `knobs_example` project as well. Something similar to [this](https://github.com/earminjon/widgetbook/blob/71ee912082949746393855fa80d4ec81e6e8159e/examples/knobs_example/lib/widgetbook.dart#L144-L156).